pom.xml

```diff
@@ -225,4 +234,84 @@
 
 	<!-- (28) <profiles/> -->
 
+	    <profiles>
+        <profile>
+            <id>localhost</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <resources>
+                    <resource>
+                        <directory>src/main/config/localhost</directory>
+                    </resource>
+                </resources>
```

Comment:
This config file doesn't exist anywhere, is it missing or are these lines unnecessary?

Reply:
Good call.  I'll eliminate
Have eliminated.

Reply:
Also noted that h2 and postgres were in pom.xml twice: once in profile and once in main file.

Have kept only the version in the profile.